### PR TITLE
added package http-kernel to fix the logger interface problem

### PR DESCRIPTION
**Pull request by vikbert**
fix the following exception after setting up:
```
ClassNotFoundException in Logger.php line 23:

Attempted to load interface "LoggerInterface" from namespace "Symfony\Component\HttpKernel\Log".
Did you forget a "use" statement for "Psr\Log\LoggerInterface"?
```

### CODE REVIEW

**Comment by B-Galati** [comment]
:+1:

File: composer.json [hunk 1]
```diff
@@ -20,7 +20,8 @@
         "symfony/security": "~2.3",
         "symfony/translation": "~2.3",
         "symfony/twig-bridge": "~2.3",
-        "symfony/validator": "~2.3"
+        "symfony/validator": "~2.3",
+        "symfony/http-kernel": "^2.8"
```
Comment (lyrixx) [inline composer.json]: the list was sorted ; it's not anymore.

**Comment by Qambar** [comment]
👍

**Comment by fabpot** [comment]
I did the same but just forced 2.3 instead. Thanks.

**Pull request closed without merging**